index.js

```diff
@@ -138,6 +140,10 @@ class Conf {
 				} catch (_) {}
 			}
 
+			if (this.validate) {
+				JSON.parse(data);
+			}
```

Comment:
This is incorrect. It will never pass as it's just the same as the parse on line 147.